File: C_C++/Chu_so_tan_cung.cpp
#include <iostream>
#include <cmath>
using namespace std;

void Chu_so_tan_cung(int a, int n)
{
	unsigned long long luy_thua = pow(a,n);
	unsigned long long so_tan_cung = luy_thua % 10;
	cout << so_tan_cung;
}
int main()
{
	int a, n;
	cin >> a >> n;
	Chu_so_tan_cung(a,n);
}

File: Python/Chu_so_tan_cung.py
import math

def Chu_so_tan_cung(a,n):
	luy_thua = int(math.pow(a,n))
	so_tan_cung = int(luy_thua % 10)
	print(luy_thua)
	
a,n = map(int, input().split())
Chu_so_tan_cung(a,n)